packages/tabs/src/TabsOverflow.ts

```diff
@@ -39,14 +39,16 @@ interface TabsOverflowState {
 }
 /**
  * @element sp-tabs-overflow
+ * @slot - text label of the Tab
+ * @slot icon - The icon that appears on the left of the label
```

Comment:
I'm not sure this is in the right file. Is this documentation for the `<sp-tab>` element or the `<sp-tabs-overflow>` element?

Reply:
`<sp-tab-overflow>` also consider these two slots same as a `<sp-tab>` component. I can remove this from here if this make sense for the consumers.

Reply:
I don't see these slots in this file. Can you help me understand why you think they belong here?

Reply:
Oh yes! You are right!! slot icon and slot text are part of `Tab.ts`. Rebased now to main.

Reply:
This no longer seems to have any changes in it. Were some missing in the rebase, or can we close this now?

Reply:
Yes we can close this!!